routes/tools_routes.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const quickselect = require('quickselect');
 
 const carddb = require('../serverjs/cards');
+const cardutil = require('../dist/util/Card.js');
```

Comment:
this line is causing tests to fail, is this directory not visible when testing? is it a casing issue?